### ISSUE TITLE
Evrard Collapse problem parameters

### ISSUE THREAD

**Issue by HamidGadirov** [AMReX-Astro/Castro#2966]
Hi,

I am working with Evrard Collapse problem (https://github.com/AMReX-Astro/Castro/tree/main/Exec/gravity_tests/evrard_collapse) and wanted to ask which parameters can be changed within simulation so that to generate an ensemble (runs with different parameters so that to have variations in data).

I am especially interested in density and velocity fields. Is it possible to change parameters so that the data generated by the simulation is a bit different? If that is not possible, could you recommend other simulations from Castro?

Thanks!
Hamid

**Comment by zingale** [comment]
Hamid, what is your goal?  Are you actually interested in the Evrard problem? or do you just want something that can generate a lot of variation in the simulation?  Evrard is not something we run much and really is a test problem, so not very visually interesting.

**Comment by HamidGadirov** [comment]
No, I don't necessarily need to use the Evrard problem. I need to generate an ensemble (runs of simulation with different parameters so that there are variations in data). I need scalar and vector data, so the variations in density and velocity fields depending on the parameters are the best. Which simulation can you recommend? Thanks!
